projects/packages/search/src/customberg/components/sidebar/sidebar-options.jsx

```diff
@@ -96,16 +96,12 @@ export default function SidebarOptions() {
 					value={ trigger }
 					options={ [
 						{
-							label: __( 'Open when the user starts typing', 'jetpack-search-pkg' ),
-							value: 'immediate',
-						},
-						{
-							label: __( 'Open when results are available', 'jetpack-search-pkg' ),
-							value: 'results',
+							label: __( 'Open when user submits the form (recommended)', 'jetpack-search-pkg' ),
+							value: 'submit',
 						},
 						{
-							label: __( 'Open when user submits the form', 'jetpack-search-pkg' ),
-							value: 'submit',
+							label: __( 'Open when the user starts typing', 'jetpack-search-pkg' ),
```

Comment:
Maybe a little bit OCD, is this sentence better corresponding to [L132](https://github.com/Automattic/jetpack/pull/25393/files#diff-60b02147c4b1bb0b61b7c0999bb72a94c9bdde1714f0df9001c19cefa30ed49fR132) to be `Open when user starts typing`?

Reply:
Even though there seems to be a need for a `the` before `user`. 😄

Reply:
Good eye for detail! I agree it would be better to be consistent there.
```suggestion
							label: __( 'Open when user starts typing', 'jetpack-search-pkg' ),
```